jerry-core/lit/lit-magic-strings.cpp

```diff
@@ -53,7 +58,8 @@ lit_magic_strings_init (void)
        id = (lit_magic_string_id_t) (id + 1))
   {
     lit_magic_string_sizes[id] = lit_zt_utf8_string_size (lit_get_magic_string_utf8 (id));
-
+    lit_magic_string_lengths[id] = lit_utf8_string_length (lit_get_magic_string_utf8 (id),
```

Comment:
Perhaps, currently magic strings do not contain any Unicode characters, so their size and length should be equal.
In the case, we could just add an assertion on the equality here, and reuse the `lit_magic_string_sizes` array as array of lengths, and add a comment, describing why it is correct.